manifests/a/AtlantisWordProcessor/TheAtlantisWordProcessorTeam/4.3.7.3/AtlantisWordProcessor.TheAtlantisWordProcessorTeam.locale.en-US.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: AtlantisWordProcessor.TheAtlantisWordProcessorTeam
+PackageVersion: 4.3.7.3
+PackageLocale: en-US
+Publisher: The Atlantis Word Processor Team
+PackageName: Atlantis Word Processor
+License: Copyright (c) The Atlantis Word Processor Team

```

Comment:
A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate

Reply:
ok changed to proprietary, i'm sorry and new to this